extensions/pkg/terraformer/terraformer.go

```diff
@@ -121,8 +121,9 @@ func New(
 		logLevel:                      "info",
 		terminationGracePeriodSeconds: int64(3600),
 
-		deadlineCleaning: 20 * time.Minute,
-		deadlinePod:      20 * time.Minute,
+		deadlineCleaning:    20 * time.Minute,
+		deadlinePod:         20 * time.Minute,
+		deadlinePodCreation: 2 * time.Minute,
```

Comment:
The terraformer images are relatively large (200+ MBs).
Just to be on the safe side, should we use a bit higher timeout, for example 5 minutes?

Reply:
Done with 10698cf